README.md

```diff
@@ -31,14 +31,6 @@ The Authentication SDK is organized into components that mirror the structure of
 [API documentation](https://auth0.com/docs/auth-api).
 For example:
```

Comment:
We should remove this line as well, otherwise it will just look odd referring to a following example that doesn't exist on the page.

Reply:
29ff528a53c91a651300e81505472bd08380eede